### ISSUE TITLE
[Little improvement] Remove duplicate message 

### ISSUE THREAD

**Issue by degroote22** [yarnpkg/yarn#762]
I'd like to request a feature. IDK if it'd be hard to implement or not, but this bogged me.

I got this console warning while running 'yarn' on some project of mine.

`warning gulp > vinyl-fs > glob-watcher > gaze > globule > glob > graceful-fs@1.2.3
: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. 
Please update to graceful-fs@^4.0.0 as soon as possible.
 Use 'npm ls graceful-fs' to find it in the tree.`

Yarn seems to already give me the result of the last line command _(standard NPM error?)_  on it's first line.

**TL;DR**
Should I do this?
`Use 'npm ls graceful-fs' to find it in the tree.`

If I have this?
`gulp > vinyl-fs > glob-watcher > gaze > globule > glob > graceful-fs@1.2.3`

I'd suggest removing the last line as it may cause some unexperienced user to **look at the already given tree and think it's not correct.**

Thanks for your time.

-- edit --
I'm using Windows 10, yarn 15.1, npm 3.10.3, node 6.3.1

**Comment by kittens** [comment]
Looks like that message was included in the deprecation notice for `graceful-fs`. We aren't doing anything special here like appending that notice so there's nothing we can do to change this. Thanks for the report!
